docs/source/reference/leaderboard_faq.rst

```diff
@@ -0,0 +1,132 @@
+Orchestrator Leaderboard FAQ
+============================
+
+The orchestrator leaderboard can be found at https://explorer.livepeer.org/.
```

Comment:
Let's link to the performance table directly using the following URL param: orchestratorTable=performance

e.g. https://explorer-20-git-as-leaderboard.livepeerorg.now.sh/?orchestratorTable=performance

Reply:
Fixed in c2055f4